### PR TITLE
docs: Document @ mention in Trackers

### PR DESCRIPTION
**Pull request by Hyzual**
Part of story #41090 @ notification in tracker mass-change & artifact modal

Why?
Changed the heading formats to use simpler characters and be more explicit on which heading is bigger.
See https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Removed the bullet points in the Comments section that were completely out of proportion, they make more sense as paragraphs, really. Also, I chose to remove the hard-wrapping as it is annoying to maintain, it changes nothing to the output and we can adapt our editor windows to the width we like.

Fixed the English mistakes that I saw.

### CODE REVIEW

File: languages/en/user-guide/trackers/usage/artifact-update.rst [hunk 1]
```diff
@@ -56,55 +56,47 @@ necessary.
...
+It is common to see the project members in charge of the artifact classification and dispatch process to post the same comments again and again. Typical examples of repeatedly posted comments are: a thank you message to the submitter, a request for the submitter to provide commonly missing information like version numbers or type of machine used, etc. Rather than typing the same comments all the time, Tuleap allows project members to create a predefined set of responses. Each canned response is defined by a name and by the body of the response. To post a canned response comment, select the appropriate response from the pull down menu in the artifact update screen and submit the changes.
+
+**Notification by @ mention**
+
+While writing comments, you can mention somebody by typing ``@`` and their Tuleap username. For example: ``@admin``. After typing three or four letters, an auto-completer should appear and suggest users matching what was written after the ``@``. When you submit the comment, each mentioned user will receive an :ref:`e-mail notification <at_mention_notification>`, unless they do not have permission to see the artifact.
```
Comment (Gashmob) [inline languages/en/user-guide/trackers/usage/artifact-update.rst:80]: > an auto-completer should appear

suggestion: It's not 'it should appear' but more 'it appear'

File: languages/en/user-guide/trackers/usage/artifact-update.rst [hunk 2]
```diff
@@ -56,55 +56,47 @@ necessary.
...
-   response is defined by a name and by the body of the response.
-
-   Posting a canned response is just a matter of selecting the
-   appropriate response from the pull down menu in the artifact update
-   screen and submitting the changes.
+--------
 
-File attachments
-~~~~~~~~~~~~~~~~~
+As many comments as needed can be attached to any given artifact. Comments are
```
Comment (Gashmob) [inline languages/en/user-guide/trackers/usage/artifact-update.rst:64]: question: Why remove 'follow-up'? Comment section is still displayed as follow-ups in artifact view

**Review by Gashmob — CHANGES_REQUESTED** [review]

File: languages/en/user-guide/trackers/usage/artifact-update.rst (hunk 1 above)
Comment (Hyzual) [reply languages/en/user-guide/trackers/usage/artifact-update.rst]: You're right, done

File: languages/en/user-guide/trackers/usage/artifact-update.rst (hunk 2 above)
Comment (Gashmob) [reply languages/en/user-guide/trackers/usage/artifact-update.rst:64]: I agree, but maybe precise somewhere about this title. The user may be confused

Comment (Hyzual) [reply languages/en/user-guide/trackers/usage/artifact-update.rst]: Indeed, I had not realized the section was actually called "Follow-ups" in the UI. I changed the paragraph, is it better ?

**Review by Gashmob — APPROVED** [review]
LGTM. Wait for review from @robinsoc

**Review by robinsoc — APPROVED** [review]

**Pull request merged**